test/runtests.cmd

```diff
@@ -474,6 +475,7 @@ goto :main
 
   echo ^>^> %*
   cmd /s /c "%*"
+  if %ERRORLEVEL% NEQ 0 exit /b %ERRORLEVEL%
 

```

Comment:
`call :do` somewhat unintuitively acts like calling a new instance of the batch script with label `:do` as the entry point, from which `exit /b` will exit and return the error code as `%ERRORLEVEL%` to the context of the call site (`exit` without `/b` would have exited the entire execution environment).

Because everything about cmd scripts always makes perfect sense.